include/mbedtls/rsa.h

```diff
@@ -48,7 +48,7 @@
 #define MBEDTLS_ERR_RSA_VERIFY_FAILED                     -0x4380  /**< The PKCS#1 verification failed. */
 #define MBEDTLS_ERR_RSA_OUTPUT_TOO_LARGE                  -0x4400  /**< The output buffer for decryption is not large enough. */
 #define MBEDTLS_ERR_RSA_RNG_FAILED                        -0x4480  /**< The random generator failed to generate non-zeros. */
-#define MBEDTLS_ERR_RSA_EXPORT_UNSUPPORTED                -0x4500  /**< The requested parameter export is not possible/allowed. */
+#define MBEDTLS_ERR_RSA_UNSUPPORTED_OPERATION             -0x4500  /**< The implementation doesn't offer the requested operation, e.g. because of security violations or lack of functionality */
```

Comment:
The following error appeared in the merge commit https://github.com/ARMmbed/mbedtls/pull/1060/commits/32297e8314fde15c7119494183558f9c93d3eb86 . Both ancestors (`development` and https://github.com/ARMmbed/mbedtls/pull/1060/commits/25b96ea2daeed8f0e4a1a58540ea9c455b6678fa) pass.

```
$ ./tests/scripts/check-names.sh
Analysing source code...
1218 macros
     146 enum-consts
1018 identifiers
819 exported-symbols

Exported symbols declared in header: PASS
Names of actual-macros: PASS
Names of enum-consts: PASS
Names of identifiers: PASS
Likely typos: FAIL
MBEDTLS_ERR_RSA_EXPORT_UNSUPPORTED

Overall: FAILED
```

Reply:
@gilles-peskine-arm I intentionally subsumed the previous `EXPORT_UNSUPPORTED` under the `UNSUPPORTED_OPERATION` error code here, and the error from `check-names.sh` stems from the fact that I missed to update some documentation in `rsa.h`.